src/elements/ArtifactsTableRow/ArtifactsTableRow.js

```diff
@@ -128,7 +128,9 @@ const ArtifactsTableRow = ({
                                   <TableCell
                                     className={cellClassNames}
                                     data={
-                                      value.expandedCellContent ? value.expandedCellContent : value
+                                      value.expandedCellContent
+                                        ? { headerId: value.headerId, ...value.expandedCellContent }
+                                        : value
```

Comment:
This change adds `headerId` only for `ArtifactTableRow`. In this case, it's better to specify the `headerId` in the `generateContent` functions, in each place where we generate the content for tables.

Reply:
![Screenshot 2024-03-07 at 1 56 04 p m](https://github.com/pinis-gini-apps/ui/assets/97065270/fccd1de8-c99b-42b1-bc75-ee781212e6c0)

Reply:
the headerId is a part of value. 
in case of expandedCellContent it's removed:
 <TableCell
       data={
                   **value.expandedCellContent ? value.expandedCellContent : value**
                   }
.

one of the solutions is to add "headerId" to expandedCellContent object in createContent function.
but in this case it will be defined twice:

![Screenshot 2024-03-07 at 2 03 11 p m](https://github.com/pinis-gini-apps/ui/assets/97065270/5e592626-b5f3-4f2b-bdc4-c4b68025f41a)

Reply:
it's ok to define it twice

Reply:
done.